include/ssl_key_handler.hpp

```diff
@@ -540,9 +540,16 @@ inline std::optional<boost::asio::ssl::context> getSSLClientContext()
         return std::nullopt;
     }
 
+    // Currently remote server's certificate verfication fails for
+    // HMC provided self-signed certificates,
+    // so skip remote server's certificate verfication.
+
+    // TODO: certificate verfication can be enabled after supporting
+    // HMC-BMC connection certificate flows on both HMC and BMC.
+
     // Add a directory containing certificate authority files to be used
     // for performing verification.
-    sslCtx.set_default_verify_paths(ec);
+    /* sslCtx.set_default_verify_paths(ec);
```

Comment:
I think commenting out every line would be better for the git blame / or searching

Reply:
Yeah, like

```
//sslCtx.set_default_verify_paths(ec);
//if (ec)
//...
```